leaf-ui/rappid-extensions/LinkInspector.js

```diff
@@ -127,7 +127,7 @@ var LinkInspector = Backbone.View.extend({
         } else {
             this.$el.html(_.template(this.template)());
             $('#constant-links').val(values[0].trim());
-            this.evolvingRelations = false;
+            this.evolvingRelations = false; //redundant????
```

Comment:
Remove comment from your commit but definitely note this somewhere (maybe in a google doc?)